### ISSUE TITLE
[Programmatic access - ADF - Toggle Explorer] Incorrect name property is defined for blob containers on Toggle explorer window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3850]
**User Experience:**
User would find difficulty to identify the get the actual context of control if incorrect name property is defined for any control as user will be misguide.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1
Screen reader: Narrator, NVDA

**Note:** User credentials should NOT be included in the bug.

**Repro Steps:​**
- Launch Azure Storage Explorer app.
- Navigate and select 'Toggle Explorer' at left panel.
- Now, navigate and select any of the blob containers(vhds).
- Verify name property of blob containers(vhds).

**Actual Result:**
Incorrect name property is defined for blob containers (vhds) on Toggle explorer window. It is defined as 'vhds, tab' instead of 'vhds' and due to this, user will be misguide for that blob container.

**Expected Result:**
Name property should be defined as 'vhds' for blob containers (vhds) on Toggle explorer window so that screen reader user could get the actual context of control.

**Note:**
NVDA also announces incorrect name for vhds blob container on Toggle explorer window.

![MAS4 1 2_Blob containers](https://user-images.githubusercontent.com/72871089/101146104-a17c0b00-3640-11eb-8019-25f61a51bdfd.png)
